css-box-model/styles.css

```diff
@@ -0,0 +1,16 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+button {
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-radius: 6px;
+}
```

Comment:
You should move this code to the space marked with "Your code here" rather than making a new rule.